README.md

```diff
@@ -1,70 +1,75 @@
+    <img alt="MIT Licensed" src="https://img.shields.io/badge/license-MIT-green?style=plastic">
+  </a>
+  <a href="https://github.com/TheLuisAbarca/pokestats-react_redux">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/TheLuisAbarca/pokestats-react_redux/development?color=blue&style=plastic">
+  </a>
+  <a href="https://github.com/TheLuisAbarca/pokestats-react_redux">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/TheLuisAbarca/pokestats-react_redux?color=orange&label=%E2%98%85%20stars%20&style=plastic">
+  </a>
+  <a href="https://github.com/TheLuisAbarca">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/TheLuisAbarca?color=yellow&logo=github&style=plastic">
+  </a>
+</p>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# PokeStats
 
-## Available Scripts
+> An awesome React/Redux application that displays the stats of a Pokemon. 
+> You can filter through the list of Pokemon and see the stats of the Pokemon you choose.
+> 
 
-In the project directory, you can run:
+| Desktop version | Mobile version|
+|-----------------|---------------|
+| ![Desktop_image](./readme_imgs/desktop_ver.png) | ![Mobile_Image](./readme_imgs/mobile_ver.png) | 
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Html
+- Css
+- JavaScript
+- React
+- Redux
+- API
+- React Bootstrap
+- Jest
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[Live Demo Link](https://theluisabarca.github.io/pokestats-react_redux/)
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Explanation
+[Live Explanation](https://www.loom.com/share/7c4160ba86854344bca7eae729f481d7)
 
-### `npm run build`
+## Getting Started
+To get a local copy up and running follow these simple example steps.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+- Click on the top right green "code" button.
+- On the dropdown menu, choose "download with zip" button.
+- After download, extract the zip file and you have the project on your machine.
+- After downloading run `npm i` to install all Node packages.
+- Run `npm start` to run the project.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Author
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+👤 **Luis Abarca**
 
-### `npm run eject`
+- GitHub: [@TheLuisAbarca](https://github.com/TheLuisAbarca)
 
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **Nelson Sakwa**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

Reply:
Thanks for your kindly and useful feedback @rloterh. +1 1st_place_medal I implement the solution for this in the commit 8db61bc.